tests/scripts/thread-cert/message.py

```diff
@@ -166,6 +166,18 @@ def assertMleMessageContainsTlv(self, tlv_class_type):
                 break
 
         assert(contains_tlv == True)
+        return tlv
+
+    #To confirm if Leader contains the Route64 TLV with router_quantity assigned Router IDs.
```

Comment:
This comment should go inside the function definition. same for `last_mle_message`

Reply:
fixed